content/docs/user-guide/data-management/large-dataset-optimization.md

```diff
@@ -122,8 +122,12 @@ To make sure that the data files in the workspace are consistent with the
 
 ---
 
-> \* **copy-on-write links or "reflinks"** are a relatively new way to link
-> files in UNIX-style file systems. Unlike hardlinks or symlinks, they support
-> transparent [copy on write](https://en.wikipedia.org/wiki/Copy-on-write). This
-> means that editing a reflinked file is always safe as all the other links to
-> the file will reflect the changes.
+<admon>
+   
+\* ([copy-on-write](https://en.wikipedia.org/wiki/Copy-on-write)) links or
+**reflinks** are a type of file linking available in UNIX-style file systems.
+Unlike hardlinks or symlinks, these file entries turn into a plain copy when
```

Comment:
this is not true, file doesn't become a copy -- only certain part of the file becomes a copy 

what was the reason to touch this?

Reply:
This was [contributed](https://github.com/iterative/dvc.org/pull/4127#pullrequestreview-1190702248) by someone external who identified that the previous text was incorrect. I'm just applying it here as well (this note exists in a couple places).

Reply:
got it, yes, text is still misleading a bit ...

Reply:
I generalized the statement. PTAL